src/python_testing/TC_OCC_2_1.py

```diff
@@ -0,0 +1,254 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+#  There are CI issues to be followed up for the test cases below that implements manually controlling sensor device for 
+#  the occupancy state ON/OFF change.
+#  [TC-OCC-3.1] test procedure step 4
+#  [TC-OCC-3.2] test precedure step 3c
+
+import logging
+
+import chip.clusters as Clusters
+from matter_testing_support import MatterBaseTest, TestStep, async_test_body, default_matter_test_main
+from mobly import asserts
+
+
+class TC_OCC_2_1(MatterBaseTest):
+    async def read_occ_attribute_expect_success(self, endpoint, attribute):
+        cluster = Clusters.Objects.OccupancySensing
+        return await self.read_single_attribute_check_success(endpoint=endpoint, cluster=cluster, attribute=attribute)
+
+    def desc_TC_OCC_2_1(self) -> str:
+        return "[TC-OCC-2.1] Attributes with DUT as Server"
+
+    def steps_TC_OCC_2_1(self) -> list[TestStep]:
+        steps = [
+            TestStep(1, "Commissioning, already done", is_commissioning=True),
+            TestStep(2, "Read attribute list to determine supported attributes"),

```

Comment:
My test plan copy does not have this step. It starts with reading the occupancy attribute. Is there a pending PR or should we update test plans (if this is required)?

Reply:
![image](https://github.com/user-attachments/assets/34cde246-56b7-4e38-a08c-63d0527e55df)

Reply:
Will remove test step for Read attribute list to determine supported attributes

Reply:
Please add a reference for the test plan PR in the PR summary.

Reply:
Even after update, there is no step 2 for reading attribute list. Lets not make it up. You can execute it, but don't assign a step to it. It seems a prerequisite after step 1.

Reply:
Done.